### PR TITLE
added hello print statement

### PR DESCRIPTION
**Pull request by sereneys**

### CODE REVIEW

File: clean_data_sereneys.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import sys
```
Comment (hongy14) [inline clean_data_sereneys.py]: Please remove this

Comment (hongy14) [reply clean_data_sereneys.py]: please remove line 1.

Comment (hongy14) [inline clean_data_sereneys.py:1]: Please remove this.

**Review by hongy14 — CHANGES_REQUESTED** [review]
Please remove line 1.
Comment update.

**Review by ylamhash — COMMENTED** [review]
Looks good ladies!

**Pull request merged**